azure-functions-extension-blob/setup.py

```diff
@@ -3,6 +3,21 @@
 
 from setuptools import setup, find_packages
 
+# TODO: pin to ext base version after published
+
+EXTRA_REQUIRES = {
+    'dev': [
+        'flake8~=4.0.1',
```

Comment:
We should move to pyproject.toml. setuptools is deprecated for latest versions of python

Reply:
updated